packages/firestore/exp/src/api/database.ts

```diff
@@ -169,6 +170,7 @@ export function enableIndexedDbPersistence(
   firestore: FirebaseFirestore,
```

Comment:
Is compat `FirebaseFirestore` compatible with the exp `FirebaseFirestore`? Or should the type be something like`FirebaseFirestore | FirebaseFirestoreCompat`?

Reply:
I don't want to pollute the generated types if we do ever merge https://github.com/firebase/firebase-js-sdk/pull/3790